docs/CHANGELOG.rst

```diff
@@ -28,6 +29,8 @@ Fixed
 -----
 * Enable DEBUG setting in tests using Django's LiveServerTestCase
 * Wait for ElasticSeach to index the data in KBBioProcessTestCase
+* Add RSEM genome index files for testing
+* Modify ``prepare_genome`` helper function to accept more than one genome
```

Comment:
This entry can be misleading. The helper function accepts a single user-defined or custom genome.